.claude/commands/plan-work.md

```
@@ -0,0 +1,19 @@
+Invoke the product-owner agent to plan and propose tasks for a goal.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)